ios/RCTTWVideoModule.m

```diff
@@ -108,9 +108,6 @@ - (dispatch_queue_t)methodQueue {
 
 - (void)addLocalView:(TVIVideoView *)view {
   [self.localVideoTrack addRenderer:view];
-  if (self.camera && self.camera.device.position == AVCaptureDevicePositionFront) {
```

Comment:
I think you'll need to keep this logic around in case the local view is added after the local video view is started.

Better yet consider refactoring this into a function like setLocalViewMirror.

Reply:
@slycoder I made the change to keep it. I don't know how to refactor it into `setLocalViewMirror`